pkg/parser/parser.go

```diff
@@ -12,7 +13,11 @@ type Parser struct {
 
 func New(query string) *Parser {
 	input := antlr.NewInputStream(query)
-	lexer := fql.NewFqlLexer(input)
+	// converts tokens to upper case, so now it doesn’t matter
```

Comment:
To upper case?

Reply:
Yes. You can type key-words in any case because they all will be converted to upper case before parsing.